miqa/learning/nn_classifier.py

```diff
@@ -314,32 +314,33 @@ def evaluateMany(model, image_paths):
     )
     device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
 
-    results = {}
-    for image_path in image_paths:
-        evaluation_ds = monai.data.Dataset(
-            data=[
-                {
-                    'img': image_path,
-                    'info': torch.FloatTensor([0] * (regression_count + len(artifacts))),
-                }
-            ],
-            transform=train_transforms,
-        )
-        evaluation_loader = DataLoader(
-            evaluation_ds, batch_size=1, pin_memory=torch.cuda.is_available()
-        )
+    evaluation_ds = monai.data.Dataset(
```

Comment:
The previous version was closer to working. Let me write this function. I will push it to this fork.

Reply:
Ah okay, thanks. I was a bit confused about what you had in mind; I'm not familiar with the usage of these monai and torch classes.